firmware/Core/Src/telecommands/telecommand_args_helpers.c

```diff
@@ -83,6 +83,86 @@ uint8_t TCMD_extract_uint64_arg(const char *str, uint32_t str_len, uint8_t arg_i
     return 0;
 }
 
+/// @brief Extracts the longest substring of integer characters, starting from the beginning of the
```

Comment:
String parsing functions on satellite missions should not do "best attempt at parsing the longest valid response". They should know definitively if the string is fully valid or not. 

If we sent the `turn_off_forever(189000O000`) telecommand, where the arg is the timestamp to turn off forever, do we want the parser to extract `189000` (the longest numerical string from the start) because there's a typo with a letter `O` in the string? Or do we want it to say "I think you made a typo; try again"?

Reply:
Oops! I did write the function that way, but I forgot to fix the comment. Good catch -- thanks!